client/components/boards/boardsList.jade

```diff
@@ -26,8 +26,6 @@ template(name="boardList")
 
 
 template(name="boardListHeaderBar")
-  h1 {{_ 'my-boards'}}
```

Comment:
I think you unexpectedly removed this title on the home page (or is there a reason?)